rakelib/artifacts.rake

```diff
@@ -638,6 +639,8 @@ namespace "artifact" do
         out.config_files << "/etc/logstash/pipelines.yml"
         out.config_files << "/lib/systemd/system/logstash.service"
         out.config_files << "/etc/default/logstash"
+        out.conflicts << "logstash (<< 7.10.0)"
```

Comment:
We may wanna explore using Breaks here, as it may be more appropriate, as per the Debian policy.